Repo/TodoRepo.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.AspNetCore.Mvc;
+using todoDotNet6.Repo;
+
+namespace todoDotNet6.TodoRepo
+{
+
+    public interface ITodoRepository : IRepository<Todo,Guid> {}
+
+    public class TodoRepo : ControllerBase, ITodoRepository
+    {
+        public static Dictionary<Guid, Todo> todos = new Dictionary<Guid, Todo>() {};
+
+        public Todo CreateTodo(Todo request)
+        {
+            request.Id = Guid.NewGuid();
+            request.CreatedDate = DateTime.Now;
+            todos.Add(request.Id, request);
+            return request;  
+        }
+
+        public Dictionary<Guid, Todo> DeleteTodo(Guid id)
```

Comment:
Why does the calling code:

* Need a dictionary of todos?
* Need to know that the repo uses a dictionary internally?

Exposing implementation detail like this violates encapsulation and makes the abstraction leaky.